app/__init__.py

```diff
@@ -96,4 +97,13 @@ def set_request_id():
 
     payload_tracker.init_payload_tracker(app_config, producer=payload_tracker_producer)
 
+    # HTTP request metrics
+    if config_name != "testing":
+        PrometheusMetrics(
+            flask_app,
+            defaults_prefix="inventory",
+            group_by="url_rule",
+            excluded_paths=["^/metrics$", "^/health$", "^/version$", "^/favicon.ico$"],
```

Comment:
This is a regex, so the period in `^/favicon.ico$` should be escaped.
```suggestion
            excluded_paths=["^/metrics$", "^/health$", "^/version$", "^/favicon\\.ico$"],
```